attestor/command/sign.go

```diff
@@ -94,6 +97,11 @@ func runSign() error {
 		return fmt.Errorf("note name is invalid %v", err)
 	}
 
+	// Silence glog outputs from kritis-signer
+	flag.Parse()
+	flag.Lookup("logtostderr").Value.Set("true")
```

Comment:
Miraculously, `Lookup()` returns non-nil even when that CLI flag isn't explicitly set. However, I don't know the exact conditions it would return nil, which makes me a bit nervous, and so I'm removing this snippet.

Reply:
> Miraculously, Lookup() returns non-nil even when that CLI flag isn't explicitly set.

I believe it will return non-nil when the flag isn't set. But will return nil if the flag doesn't exist.